go/vt/vttablet/tabletserver/tabletserver.go

```diff
@@ -192,6 +192,10 @@ type TabletServer struct {
 
 	// alias is used for identifying this tabletserver in healthcheck responses.
 	alias topodatapb.TabletAlias
+
+	// experimental flag that makes it possible
+	// to create read only transactions on non master tablets
+	allowReadOnlyTxOnNonMaster bool
```

Comment:
We don't need this flag. Let's always open `te`. In fact, we should move the call to `fullStart`. This will eventually allow us to look at replica transactions in the future.

Reply:
I like it but does this introduce a potential split brain issue for environments that exclusively use Vitess to do reparenting? Or do we enforce single master at some other level?